native/src/routes/search/components/SearchModal.js

```diff
@@ -123,6 +128,8 @@ class SearchModal extends React.Component<PropsType, StateType> {
     }
 
     const filteredCategories = this.findCategories(categories)
+    const feedbackOrigin = filteredCategories.length !== 0 ? 'searchInformationNotFound' : 'searchNothingFound'
```

Comment:
According to some article I saw a while ago negated expressions are harder to understand than non-negated things. I don't mind keeping it this way though, just wanted to share this.

Reply:
you are right. changed it.